nucypher/characters/control/interfaces.py

```diff
@@ -107,17 +107,17 @@ def grant(self,
     def revoke(self, label: bytes, bob_verifying_key: bytes) -> dict:
 
         # TODO: Move deeper into characters
-        policy_id = construct_policy_id(label, bob_verifying_key)
-        policy = self.implementer.active_policies[policy_id]
+        policy_hrac = HRAC.derive(self.implementer.stamp.as_umbral_pubkey(), bob_verifying_key, label)
+        policy = self.implementer.active_policies[policy_hrac]
```

Comment:
An issue for another PR, but I don't think this active policy cache is used anywhere.

Reply:
It is used once in `AliceInterface.revoke()`